tokio/src/task/mod.rs

```diff
@@ -304,4 +304,9 @@ cfg_rt! {
         mod builder;
         pub use builder::Builder;
     }
+
+    /// Task-related futures.
+    pub mod future {
```

Comment:
The module added by #3840 is named `futures`. https://docs.rs/tokio/1.8.0/tokio/sync/futures/index.html
It would be preferable for them to have the same name.

Reply:
Good catch.

Reply:
```suggestion
    pub mod futures {
```